tests/js/spec/utils/formatters.spec.jsx

```diff
@@ -32,6 +35,9 @@ describe('getDuration()', function () {
     expect(getDuration(86400, 0, true)).toBe('24hr');
     expect(getDuration(86400 * 2, 0, true)).toBe('2d');
     expect(getDuration(604800, 0, true)).toBe('1wk');
+    expect(getDuration(604800 * 2, 0, true)).toBe('2wk');
+    expect(getDuration(2629800, 0, true)).toBe('1mos');
```

Comment:
Shouldn't this be `mo` as there is only one?

Reply:
@markstory good point, it does look like Mo. is used for singular